exporter/awss3exporter/marshaler.go

```diff
@@ -31,6 +31,12 @@ func NewMarshaler(mType MarshalerType, logger *zap.Logger) (marshaler, error) {
 		marshaler.tracesMarshaler = &ptrace.JSONMarshaler{}
 		marshaler.metricsMarshaler = &pmetric.JSONMarshaler{}
 		marshaler.fileFormat = "json"
+	case SumoIC:
+		sumomarshaler := NewSumoMarshaler()
+		marshaler.logsMarshaler = &sumomarshaler
+		marshaler.tracesMarshaler = &sumomarshaler
+		marshaler.metricsMarshaler = &sumomarshaler
```

Comment:
why set it if unsupported?

Reply:
It doesn't matter as the factory returns an error. I can change this for clarity.